Documentation/config.txt

```diff
@@ -1468,7 +1468,7 @@ commit.verbose::
 	See linkgit:git-commit[1].
 
 commit.allowEmpty::
-	A boolean to specify weather empty commits are allowed with `git commit`.
+	A boolean to specify whether empty commits are allowed with `git commit`.
```

Comment:
Ah, see, I commented on the first patch, and spent brain cycles... It is much better to squash the change into the first commit, avoiding to confuse the reviewers (which might even include you, 6 months from now, when you have to look up something and stumble across your own commit message).